minari/data_collector/data_collector.py

```diff
@@ -250,8 +265,8 @@ def reset(
         assert STEP_DATA_KEYS.issubset(step_data.keys())
 
         # If last episode in global buffer has saved steps, we need to check if it was truncated or terminated
-        # If not, then we need to auto-truncate the episode
-        if len(self._buffer[-1]["actions"]) > 0:
+        # If not (empty dicitionary), then we need to auto-truncate the episode.
+        if self._buffer[-1]:
```

Comment:
Feel like there's probably a cleaner way to test for this but I'd have to look into the code a bit more deeply to be sure

Reply:
I'm not sure, but I'm open to suggestions here.